mode.c

```diff
@@ -224,6 +224,7 @@ void init_vm_mode(void) {
 	behavior_start(B_LEDS_PROX);
 	behavior_start(B_SOUND_BUTTON);
 	behavior_start(B_LEDS_MIC);
+	prox_disable_network();
```

Comment:
But in which case it is needed ? the MODE_FOLLOW is the only one enabling the network, and it is properly disabled at mode exit. Thus if it is really needed, it is papering over a bug elsewere.

We should better find were the network mode should be disabled (or understand why it need to be done here and explain in a comment).